config.json

```diff
@@ -7,7 +7,7 @@
       "reviewers": ["olehermanse", "vpodzime"],
       "jenkins": "https://ci.cfengine.com/",
       "jenkins_job": "pr-pipeline",
-      "bot_features": ["create_pr_magic", "update_dependencies", "generate_changelogs", "ping_reviewer"],
+      "bot_features": ["create_pr_magic", "update_dependencies", "generate_changelogs", "ping_reviewer", "tag_builds"],
```

Comment:
@Lex-2008 This config is for the instance of tom running on jenkins machine. I believe it should not have any of the slack features, since you are running this elsewhere?